test/unit/org/apache/cassandra/cql3/statements/DescribeStatementTest.java

```diff
@@ -757,6 +757,103 @@ public void testDescribeWithCustomIndex() throws Throwable
                       row(KEYSPACE_PER_TEST, "index", indexWithOptions, expectedIndexStmtWithOptions));
     }
 
+    @Test
+    public void testUsingReservedInCreateType() throws Throwable
```

Comment:
nit: maybe call it `testDescTypeShouldNotOmitQuotations` or similar. Similarly, change the names of the other newly added test cases.

The reason is that the field names are quoted and they are not really "reserved". The test is to verify the output of `DESC` should preserve the quotation marks.